Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -255,6 +256,7 @@ class UIProxyForCapture final : public UserMediaCaptureManagerProxy::ConnectionP
 WebProcessProxy::WebProcessProxy(WebProcessPool& processPool, WebsiteDataStore* websiteDataStore, IsPrewarmed isPrewarmed, CrossOriginMode crossOriginMode, LockdownMode lockdownMode)
     : AuxiliaryProcessProxy(processPool.alwaysRunsAtBackgroundPriority())
     , m_backgroundResponsivenessTimer(*this)
+    , m_periodicalResponsivenessTimer(makeUnique<PeriodicalResponsivenessTimer>(*this))
```

Comment:
Should this only be initialized on GTK/WPE ports? I don't think we want this for Cocoa ports for example.